logs/audit-index.csv

```diff
@@ -13,3 +13,7 @@ ce75011f34cd32746660387a510a8090c3c3a60a, 2022-03-02T21:15:43+00:00
 f7824a0db868c6fb35dfa1406619bf340e25c144, 2022-03-09T23:04:47+00:00
 4e17ef73a670f373b56917ce506ebedc7fa0cb29, 2022-03-09T23:42:57+00:00
 dc886a9ad1beed821bf1305bd364b3865744ab9d,2022-03-11T12:17:06+00:00
+a5437cec13d0cb9ceeba6e02c250a069194f8b75,2022-03-12T04:56:37+00:00
```

Comment:
Do you want to add a space between ID and timestamp, as was added in the yaml file.